test/connection-checker/src/net.rs

```diff
@@ -74,3 +75,53 @@ pub fn send_ping(opt: &Opt, destination: IpAddr) -> eyre::Result<()> {
 
     Ok(())
 }
+
+#[cfg(target_os = "macos")]
+pub fn send_ping(opt: &Opt, destination: IpAddr) -> eyre::Result<()> {
```

Comment:
maybe add a comment explaining why this function is different from the windows one

Reply:
Done!